docs/controller-skills/military.md

```diff
@@ -126,4 +126,13 @@ MARSA is to be assigned and terminated on pilots request, and when standard sepe
 - Flights that are on nominated route segments, except for short position reports when possible.
 
 !!! note
-    Military aircraft normally operate NOCOM in military Restricted Areas and Class G airspace only. **Do not** permit NOCOM operations in Civil CTA.
\ No newline at end of file
+    Military aircraft normally operate NOCOM in military Restricted Areas and Class G airspace only. **Do not** permit NOCOM operations in Civil CTA.
+
+## Stereo Approaches
+Stereo Approaches are the primary procedure for Fast Jet Aircraft to recover back to fast jet airbases such as Willy and Tindal. The approach consits of Fast Jet tracking to an exit gate of the SUA and then following each airports stereo procedure onto final in preperation for an initial and pitch or instrument approach.
```

Comment:
Try to stick to using ICAO codes (YWLM, YPTN) rather than airport names or nicknames (Willy, Tindal). Additionally, just include an example of a few Fast Jet aircraft. Added some spelling/punctuation suggestions too

```suggestion
Stereo Approaches are the primary procedure for Fast Jet Aircraft (eg, **F18**, **HAWK**) to recover back to fast jet airbases such as YWLM and YPTN. The approach consists of Fast Jet tracking to an exit gate of the SUA, and then following each airports stereo procedure onto final in preparation for an initial and pitch, or instrument approach.
```